src/hb-aat-layout.cc

```diff
@@ -46,11 +46,16 @@
  * @short_description: Apple Advanced Typography Layout
  * @include: hb-aat.h
  *
- * Functions for querying OpenType Layout features in the font face.
+ * Functions for querying OpenType Layout features in the font face. 
+ *
+ * HarfBuzz supports all of the AAT tables used to implement shaping. Other
+ * AAT tables and their associated features are not supported.
```

Comment:
HarfBuzz doesn't support `just` which is related to shaping but supports `lcar`, `feat` and `sbix`. There are of course tables that are not hooked yet like `fdsc` (which Apple itself don't care about, we may remove it also I guess) and `bsln` (`BASE` also isn't yet)

Reply:
`just` isn't in that use in Apple's fonts also of course.

Reply:
I get that wording from Behdad.... `lcar` seems to fall under the same purview as I see it; reading it provides compatibility with what we say about OpenType shaping being important for caret placement.

Also, I think "supported" is important here -- AIUI, we're just wanting to declare that the purpose is to support shaping; anything else is not be considered a guarantee. (sbix being the domain of hb-ot-color, etc). 

Do you (or @behdad ) have some other wording in mind you would recommend for that sentence?

Reply:
Given your explanation I feel I understand the intention of course, this corner case didn't make sense just at first so may change of wording is needed or not.

Reply:
> anything else is not be considered a guarantee

I think we like to work on any features AAT has and we don't as I was involved in `lcar` and `sbix` and like to make it on par with OT. May it isn't guaranteed, like the rest of the project, but I think we like to get feedback on not supported things anytime, like feature setting using AAT's convention, that may fall into shaping scope actually. This is anyway some kind of bike-shedding at best so sorry for taking your time.

Reply:
No problem at all. I do think we ought to be clear, but perhaps this section reference is not the ideal spot. Perhaps we can add more detail about what is accessible in the 'user manual' page on AAT.